code/index.html

```diff
@@ -1,30 +1,162 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Project 3 - Pour Over Purist</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Playfair+Display:900|Rosario&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+  <!-- video or image as a header is cool :) -->
+
+  <header>
+    <section class="video-container">
+
+      <video class="mobile-video" autoplay muted loop playsinline>
+        <source src="videos/video-mobile.mp4" type="video/mp4">
+        <p>Your browser does not support the video tag.</p>
+      </video>
+
+      <video class="tablet-video" autoplay muted loop playsinline>
+        <source src="videos/video-tablet.mp4" type="video/mp4">
+        <p>Your browser does not support the video tag.</p>
+      </video>
+
+      <video class="desktop-video" autoplay muted loop playsinline>
+        <source src="videos/video-desktop.mp4" type="video/mp4">
+        <p>Your browser does not support the video tag.</p>
+      </video>
+
+      <div class="video-overlay">
+        <h1 class="overlay-heading">Pour Over Purist</h1>
+        <p class="overlay-paragraph">
+          Online courses for wannabe percolation perfectionsts
+        </p>
+        <a class="get-started" href="#sign-up">Get started</a>
+      </div>
+
+    </section>
+  </header>
+
+  <section class="main-container">
     <!-- Signup form -->
 
+    <section class="sign-up" id="sign-up">
+      <form class="contact-form" action="https://httpbin.org/anything" method="POST">
+        <h2>Sign Up!</h2>
+
+        <label>Name
+          <input type="text" name="name" placeholder="Enter your name" required>
+        </label>
+
+        <label>Email
+          <input type="text" name="email" placeholder="Enter your email address" required>
```

Comment:
There's a type called email actually! 
```suggestion
          <input type="email" name="email" placeholder="Enter your email address" required>
```

Reply:
Oh, missed that one! Have updated it ☺